src/raftstore/errors.rs

```diff
@@ -188,13 +188,54 @@ impl Into<errorpb::Error> for Error {
                 errorpb.set_stale_command(errorpb::StaleCommand::new());
             }
             Error::Transport(transport::Error::Discard(_)) => {
-                let mut server_is_busy_err = errorpb::ServerIsBusy::new();
-                server_is_busy_err.set_reason(RAFTSTORE_IS_BUSY.to_owned());
-                errorpb.set_server_is_busy(server_is_busy_err);
+                let mut reason = errorpb::ServerIsBusy::new();
+                reason.set_reason(RAFTSTORE_IS_BUSY.to_owned());
+                errorpb.set_server_is_busy(reason);
             }
             _ => {}
         };
 
         errorpb
     }
 }
+
+impl From<errorpb::Error> for Error {
```

Comment:
This is weird.

Reply:
I want to know the precise error in tests.

Reply:
What's wrong with `has_xxx` check?